app/services/csv/case_log_csv_service.rb

```diff
@@ -0,0 +1,90 @@
+module Csv
+  class CaseLogCsvService
+    CSV_FIELDS_TO_OMIT = %w[hhmemb net_income_value_check sale_or_letting first_time_property_let_as_social_housing renttype needstype postcode_known is_la_inferred totchild totelder totadult net_income_known is_carehome previous_la_known is_previous_la_inferred age1_known age2_known age3_known age4_known age5_known age6_known age7_known age8_known letting_allocation_unknown details_known_2 details_known_3 details_known_4 details_known_5 details_known_6 details_known_7 details_known_8 rent_type wrent wscharge wpschrge wsupchrg wtcharge wtshortfall rent_value_check old_form_id old_id retirement_value_check tshortfall_known pregnancy_value_check hhtype new_old vacdays].freeze
+
+    def initialize(user)
```

Comment:
Currently this would export all logs right? Shouldn't it rather take a collection as an input?

Reply:
_.all_ returns already filtered logs

Reply:
It does if you call it on a collection, but if you call it on the class (as on line 14) what is it filtering by?

Reply:
I mean from what I can see, it does filter it out, I'm happy to pass it in, but I'd be passing in the same thing, because that's what was being used in the to_csv method to begin with (I think ??)

Reply:
Yep you're right it does.